components/script/dom/htmlscriptelement.rs

```diff
@@ -450,6 +456,46 @@ impl HTMLScriptElement {
         }
     }
 
+    fn unminify_js(&self, script: &mut ClassicScript) {
+        if !opts::get().unminify_js {
+            return;
+        }
+
+        let process = Command::new("js-beautify")
+                              .stdin(Stdio::piped())
+                              .stdout(Stdio::piped())
+                              .spawn()
+                              .expect("Failed to execute js-beautify");
```

Comment:
Instead of making this a fatal error, let's dump the unmodified script into the file if js-beautify isn't available and leave an informative debug!.